Treasure hunt (Treasure and player settings)

ChielBruin opened the pull request:
- Added a treasure to each level.
- Reduced player jump and walk speed
- Increased player health.
- Fixed a bug where entities could go through walls in a lag-spike.
- Fixed a bug where the bomb timer was reset when picked up.

Note: This PR should be merged after #335 is closed.

ChielBruin committed:
Added treasure spawning

ChielBruin committed:
Improved player settings

Decreased jumping height
Decreased movement speed

ChielBruin committed:
Changed the random light to a white light.

ChielBruin committed:
Increased the number of rooms in each level

ChielBruin committed:
Removed files

ChielBruin committed:
Fixed player spawning height

ChielBruin committed:
Increased player health to 5

ChielBruin committed:
Merge remote-tracking branch 'origin/feature-hud' into feature-demo-fixes

ChielBruin committed:
Fixed failing tests

ChielBruin committed:
Merge branch 'feature-hud' into feature-demo-fixes

WytzeElhorst commented:
The .blend1 files it adds are old backups that blender automatically creates, so they can be deleted

ChielBruin committed:
Merge branch 'feature-hud' into feature-demo-fixes

ChielBruin committed:
Fixed a bomb bug

The timer got a reset when the bomb was picked up.

ChielBruin committed:
Fixed physics clipping entities

Entities were able to glitch through walls when a big leg-spike occured.
This was caused by a very high tpf that made the entity move through a
wall before colliding with it.

ChielBruin committed:
Fixed a failing test

Diocruel commented:
Can be merged.

ChielBruin committed:
Merge branch 'dev-sprint6' into feature-demo-fixes

Geweldig commented:
Will merge once travis is done.
